lib/msf/core/exploit/auto_check.rb

```diff
@@ -27,17 +27,25 @@ def exploit
     case (checkcode = check)
     when Exploit::CheckCode::Vulnerable, Exploit::CheckCode::Appears
       print_good(checkcode.message)
+      return super
     when Exploit::CheckCode::Detected
       print_warning(checkcode.message)
+      return super
     when Exploit::CheckCode::Safe
+      print_warning(checkcode.message)
+      return super if datastore['ForceExploit']
       fail_with(Module::Failure::NotVulnerable,
-                "#{checkcode.message} Disable AutoCheck to override.")
+                "#{checkcode.message} Disable ForceExploit to override.")
```

Comment:
I think this is _exactly_ what I was trying to go for when I made the `AutoCheck` mixin, but I'm a big dummy and tried to destroy `ForceExploit` instead. 😅 ❤️

Reply:
Corrected in https://github.com/adfoster-r7/metasploit-framework/pull/1, btw.